examples/react-apollo/src/components/email-passwordless-form.tsx

```diff
@@ -6,9 +6,13 @@ import React, { useState, useEffect } from 'react'
 export const EmailPasswordlessForm: React.FC = () => {
   const [email, setEmail] = useState('')
   const navigate = useNavigate()
-  const { signInEmailPasswordless, isError, isSuccess, error } = useSignInEmailPasswordless(email, {
-    redirectTo: '/profile'
-  })
+  const { signInEmailPasswordless, isError, isSuccess, error } = useSignInEmailPasswordless(
+    // TODO correct this once the new packages are released
+    undefined,
+    {
+      redirectTo: '/profile'
+    }
+  )
   const [showError, setShowError] = useState(true)
```

Comment:
is this `showError` needed?

Reply:
yes, this is required to hide the error message when the user changes the email field

Reply:
Ok, I would maybe not bother with that but I see your point.